### PR TITLE
Bta 571: View, Check or Correct Submissions

### PR DESCRIPTION
**Pull request by AndrewShanks**

### CODE REVIEW

File: test/views/epaye/ViewCheckCorrectSubmissionsSpec.scala [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  def docWithoutEmail = asDocument(createViewWithoutEmail())
+
+  "View Check Correct Submissions view with an email accessible" must {
+
+    behave like normalPage(createView, messageKeyPrefix)
+
+    "contain correct content" in {
+
+      doc.getElementsByTag("h1").first().text() mustBe "View, check or correct your submissions"
+      doc.text() must include("Submissions show up in your balance at different times depending on when you submit them.")
```
Comment (gianick) [inline test/views/epaye/ViewCheckCorrectSubmissionsSpec.scala]: Please remove tests that check non-dynamic content. Thank you :)

File: test/controllers/HelpAndContactControllerSpec.scala [hunk 2]
```diff
@@ -31,7 +31,7 @@ import views.html.vat._
 
 class HelpAndContactControllerSpec extends ControllerSpecBase {
 
-  def fakeServiceInfoRequest(utr: Option[SaUtr] = None) = ServiceInfoRequest(AuthenticatedRequest(fakeRequest, utr), HtmlFormat.empty)
+  def fakeServiceInfoRequest(utr: Option[SaUtr] = None) = ServiceInfoRequest(AuthenticatedRequest(fakeRequest, utr, Some("user@test.com")), HtmlFormat.empty)
```
Comment (gianick) [inline test/controllers/HelpAndContactControllerSpec.scala]: Maybe use 'example.com'?

File: test/views/epaye/ViewCheckCorrectSubmissionsSpec.scala [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+      docWithoutEmail.text() must include("If you use commercial software to manage your payroll, check with your software provider whether it can process Earlier Year Updates.")
+      docWithoutEmail.text() must include("If your software doesn’t offer Earlier Year Updates, you will need to")
+    }
+
+    "have the correct 'Add an email' link" in {
+      assertLinkById(docWithoutEmail,"add-email", "add an email",
+        "http://localhost:9020/business-account/manage-account/government-gateway-warning",
+        "HelpEPAYEContentLink:click:AddEmail", expectedOpensInNewTab = false)
+    }
+
```
Comment (gianick) [inline test/views/epaye/ViewCheckCorrectSubmissionsSpec.scala]: Please remove unnecessary line breaks. Thank you :)

**Comment by Shaju-11** [comment]
:+1:

**Pull request merged**